setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# W391 blank line at end of file
+# E302 expected 2 blank lines, found 1
+ignore = W391,E302
```

Comment:
Yeah, our workflow isn't the typical everyone-commits-directly-to-repo workflow, so we could make it work.